Formula/a/arxiv_latex_cleaner.rb

```diff
@@ -16,20 +18,36 @@ class ArxivLatexCleaner < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "848d878cb7be53a4cb28b29faf3d79af0d9698aa43d2073d1af1ba2fb83691ed"
   end
 
-  depends_on "python-setuptools" => :build
-  depends_on "pillow"
-  depends_on "python-abseil"
-  depends_on "python-regex"
+  # jpeg is needed to build pillow
+  depends_on "jpeg" => :build
```

Comment:
From what I can tell, a libjpeg is required only for building the extension. The `pillow` wheel shipped to pypi for example include standalone dylibs.

Reply:
However, should probably still verify that the functionality that depends on a jpeg lib actually works in tests? I'm not yet convinced that the pip install done into the virtualenv here will build the standalone dylibs.

Reply:
(followup) it was easy enough for me to verify with `mapproxy` that there is the runtime dependency on the jpeg library, so probably keeping the dependency for both build and runtime makes a lot of sense here.